src/lib/style/components/_cards.scss

```diff
@@ -15,19 +15,80 @@ $text-color-lighter: #898989;
   font-size: $text-size-base;
 }
 
-// TODO: Simplify and remove this scss, when we move to a single
-// card type there should be no special casing as is below
 .SRC-portalCard {
```

Comment:
Merge the .SRC-portalCard and .SRC-portalCardHeader definitions.  Now SRC-portalCardHeader overrides a few default styles defined in SRC-portalCard.